README.md

```diff
@@ -51,4 +51,37 @@ Cross team meetings
 Meeting notes from the cross team meetings can be found in [Cross Team](Cross%20Team/README.md).
 
 
+Presentations
+--------------------------
+
+|    Date      |        Title                      |    Venue/Audience    |    Presenter     |   Link  to material/ announcement     |
+|:-------------|:---------------------------------:|:--------------------:|:----------------:|:-------------------------------------:|
+|  2022-06-13  |Introduction to ESMValTool: webinar| IS-ENES3             |  Klaus           |  https://is.enes.org/events/introduction-to-esmvaltool-a-tool-for-evaluation-of-climate-model-data-webinar-and-workshop|
```

Comment:
it might be nice to add github usernames, so it is a bit more clear who is who to external readers. For example:
```suggestion
|  2022-06-13  |Introduction to ESMValTool: webinar| IS-ENES3             |  Klaus [@zklaus](https://github.com/zklaus)          |  https://is.enes.org/events/introduction-to-esmvaltool-a-tool-for-evaluation-of-climate-model-data-webinar-and-workshop|
```

Reply:
@bouweandela thanks, I added usernames for all.